api-doc/api.raml

```diff
@@ -0,0 +1,18 @@
+#%RAML 1.0
+title: eLife's Libero walking skeleton
+
+baseUri: http://localhost:8085
+
+resourceTypes:
+    articles: !include articles.raml
+
+types:
+    ArticleVersionFront:
+        type: !include http://api-dummy-nginx:80/schemas/elife/article/front.rng # don't work as contains localhost references
```

Comment:
Wouldn't normally be a problem, but for now `localhost:8086` will need to be proxied to `schemas:80` somehow.

Reply:
are these references meant to be moved to `schemas:80`? `api-dummy-nginx:80` seems to have the same problems

Reply:
I see the `Can not resolve http://api-dummy-nginx:80/articles/00666/latest/front` now